sources/packages/backend/libs/utilities/src/address-utils.ts

```diff
@@ -0,0 +1,17 @@
+import { AddressInfo } from "@sims/sims-db";
+
+// 'selectedCountry' in the student contact info will have the value 'other',
+// when 'Other'(i.e country other than canada) is selected.
+export const OTHER_COUNTRY = "other";
+// 'selectedCountry' in the student contact info will have the value 'canada',
+// when 'Canada' is selected.
+export const COUNTRY_CANADA = "canada";
+
+/**
+ * Inspects the address info to determine if the address is from Canada.
+ * @param address address info to be inspected.
+ * @returns true if the address is from Canada, otherwise false.
+ */
+export function isAddressFromCanada(address: AddressInfo) {
+  return address.selectedCountry === COUNTRY_CANADA;
```

Comment:
We are using just the `selectedCountry` to determine if the address is from Canada. Do we need the whole object passed here?

Reply:
From the PR description. Please let me know if it is not clear
![image](https://github.com/user-attachments/assets/c9ccc61c-4dda-429c-a5c1-b48507c504a3)

Reply:
Apologies, missed it. Thanks.